lib/oneview-sdk/file_upload_helper.rb

```diff
@@ -0,0 +1,90 @@
+# (C) Copyright 2016 Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+require_relative 'resource'
+
+module OneviewSDK
+  # Helper for upload of files
+  module FileUploadHelper
```

Comment:
Actually, what about adding this to the client class, like we do for [rest.rb](https://github.com/HewlettPackard/oneview-sdk-ruby/blob/master/lib/oneview-sdk/rest.rb)? We can just [include it ](https://github.com/HewlettPackard/oneview-sdk-ruby/blob/master/lib/oneview-sdk/client.rb#L24), so then we could do `client.download_file` and `client.upload_file`.